src/main/kotlin/step3/domain/RaceManager.kt

```diff
@@ -0,0 +1,28 @@
+package step3.domain
+
+/**
+ *
+ * @author Leo
+ */
+class RaceManager {
+
+    fun getParticipants(participantCount: Int): List<Car> {
+        val forwardStrategy = ForwardImpl()
+        val cars = mutableListOf<Car>()
+        for (num in 1..participantCount) {
+            cars.add(Car(forwardStrategy))
+        }
+
+        return cars
+    }
```

Comment:
단순히 개수에 따른 Car를 만드는 것이라면, 아래 처럼도 쓸 수 있겠네요!
```suggestion
    fun getParticipants(participantCount: Int): List<Car> {
        val forwardStrategy = ForwardImpl()
        return MutableList(participantCount) { Car(forwardStrategy) }
    }
```
또는
```suggestion
    fun getParticipants(participantCount: Int): List<Car> {
        val forwardStrategy = ForwardImpl()
        return (1..participantCount)
            .map { Car(forwardStrategy) }
    }
```
이런식으로도 함수형 패러다임을 적용해볼 수 있습니다 🙂

Reply:
TDD 코틀린 수업 신청한 이유가 이런 가이드를 받고 싶어서였습니다! 피드백 매우 감사합니다! 반영했습니다